src/main/java/com/vmware/vipclient/i18n/base/L10nTypeEnum.java

```diff
@@ -0,0 +1,33 @@
+package com.vmware.vipclient.i18n.base;
+
+import com.vmware.vipclient.i18n.VIPCfg;
+import com.vmware.vipclient.i18n.base.cache.FormattingCache;
+import com.vmware.vipclient.i18n.base.cache.MessageCache;
+
+public enum L10nTypeEnum {
```

Comment:
Looks like this new added class is not used, so what's the purpose of adding it?

Reply:
Added this by mistake. Removing it now.